CODEOWNERS

```diff
@@ -1,18 +1,15 @@
 * @saleor/apps-guild
 /.github/ @saleor/SRE
 
-
-/apps/emails-and-messages @saleor/shopex-js
-/apps/taxes  @saleor/shopex-js
 /apps/avatax  @saleor/shopex-js
-/apps/search @saleor/shopex-js
+/apps/emails-and-messages @saleor/shopex-js
 /apps/klaviyo @saleor/shopex-js
+/apps/search @saleor/shopex-js
```

Comment:
Since we transfer ownership, lets move these to merchant:
- Emails and Messages (will be removed but for now its there)
- Klaviyo 
- Search
- SMTP